### ISSUE TITLE
Icon-only Button (without text children) is not accessible using iOS VoiceOver

### ISSUE THREAD

**Issue by kpcurai** [tamagui/tamagui#1842]
The HTML markup is correct on Web but native iOS VoiceOver can't detect or even click a Button with `aria-label` specified if there is no text content.

Example:

Enable VoiceOver from within Settings > Accessibility on iOS, and attempt to click a button like the below.

```
import { Button } from "tamagui";
import { Airplay } from "tamagui/lucide-icons";

<Button aria-label="Airplay" alignSelf="center" icon={Airplay} size="$6" />
```

**Comment by hackily** [comment]
Very interested in this

**Comment by natew** [comment]
React should support aria but I'm not sure what version

https://github.com/facebook/react-native/pull/34502

**Comment by kpcurai** [comment]
`aria-label` on a Pressable exported from react-native is sufficient to get VoiceOver to read and click icon-only buttons: however it does nothing in VoiceOver when supplied to the _Tamagui_ Button. I'm not sure from the source code--does Tamagui render a Pressable under the hood?

Probably related, the clickable region is very small within VoiceOver as the hitbox only covers the Text.

**Comment by hackily** [comment]
I'm still seeing this problem in the latest version 1.86.5, having to use a Pressable in the meantime.

**Comment by thienautran** [comment]
This is a button wide implementation error I think. I'm still seeing this issue for buttons. Mainly, if it doesn't have a text child it will never be accessible to voice over. Additionally, the button's role is not presented to voice over. 

Somewhere along the line the accessibilityRole is not being recognised, it should be "button". 

As a result, the reason for the target hitbox being so small is because the button itself is not accessible and it's actually just the text that's being accessed. 

@natew  this needs to be looked at
